content/using/install.md

```diff
@@ -24,7 +24,7 @@ To start with a comet, continue below. For planet instructions, see [further bel
 
 ## Installing Urbit
 
-Regardless of your Urbit ID, you'll want the Urbit binary installed first. The Urbit binary runs on a Unix-like operating system – Ubuntu, Fedora, or macOS, for example.
+Regardless of your Urbit ID, you'll want the Urbit binary installed first. This is done via the command line. The Urbit binary runs on a Unix-like operating system – Ubuntu, Fedora, or macOS, for example.
```

Comment:
This is a great suggestion.

Reply:
Is there a good beginners manual for using CLI that we could include? One for each OS I imagine?

Reply:
Personally feel that should be an exercise left to the reader to find.